lib/redis.js

```diff
@@ -63,7 +63,7 @@ module.exports = CoreObject.extend({
 
   _listRevisions: function(keyPrefix) {
     var client = this._client;
-    return client.lrange(keyPrefix, 0, this._maxNumberOfRecentUploads - 1);
+    return client.zrevrange(keyPrefix, 0, -1);
```

Comment:
given that we always keep the list at a `this._maxNumberOfRecentUpload` size we simplified this